### DISCUSSION TITLE
do you think modern ui can be implemented back in anytime soon?

### DISCUSSION THREAD

**Discussion by moss-ss** [cs0x7f/cstimer#424] [General]
since the cstimer+ extension doesn't work anymore, will it be possible to have the modern ui features back?

**Comment by cs0x7f** [comment]
Unfortunately, cstimer+ is not open source software, and I don't know what adjustments it makes to cstimer, so there is no way to integrate it directly into cstimer.

**Reply by moss-ss** [reply]
that's a shame, but maybe theres a way to look throughout the code and implement it?

**Comment by cs0x7f** [comment]
I tried to find some of its source code and tried to copy the ui part in: https://github.com/cs0x7f/cstimer/commit/006ed71e92e361631e16dd0ae2141ec34c1828bc
I'm not sure if the displayed ui is exactly the same as cstimer+. If you find any inconsistencies (except logos and buttons) you can take screenshots.

**Reply by moss-ss** [reply]
thanks man, looks great 🙏
